test/services/bill-runs/two-part-tariff/fetch-return-logs-for-licence.service.test.js

```diff
@@ -146,10 +146,31 @@ describe('Fetch Return Logs for Licence service', () => {
         expect(result[0].returnSubmissions[0].returnSubmissionLines).to.have.length(0)
       })
     })
+
+    describe('where the return logs start date is outside the billing period', () => {
```

Comment:
Would be good to make it clearer that the end date is within the billing period as well as the start date being outside.